Breeding-Insight/bi-api
# BI-2266 - use GID as Entry No for all germplasm export

# Description
**Story:** [BI-2266](https://breedinginsight.atlassian.net/browse/BI-2266)

The file exported through the "Download file" action on the all germplasm list view contains all germplasm in the program. When multiple germplasm lists have been uploaded, BI-2266 specifies the desired behavior is for the "Entry No" column to contain the GID (instead of a list entry number from whatever list the germplasm was originally uploaded in, which was the previous behavior).


# Testing
1. Upload multiple germplasm lists to a program.
2. Download all germplasm, ensure "Entry No" matches GID for all rows.
3. Download individual germplasm lists, ensure "Entry No" matches import entry number.


# Checklist:



[BI-2266]: https://breedinginsight.atlassian.net/browse/BI-2266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
